rules/avoid-repeated-emojis/rule.md

```diff
@@ -0,0 +1,63 @@
+---
+type: rule
+title: DRY - Do you avoid repeated emojis in lists?
+uri: avoid-repeated-emojis
+authors:
+  - title: Jack Pettit
+    url: https://www.ssw.com.au/people/jack-pettit
+related:
+  - use-emojis
+created: 2023-02-28T00:28:10.330Z
+guid: 1acd6766-d31f-493f-b6f1-3b0b2ff2dc4e
+---
+Emojis are a lot easier to consume when a lot of information is present. But when repeated excessively, they can become a hindrance rather than a help. This helps to make it more visually interesting and easier to scan quickly. In this way, emojis can enhance the communication experience, rather than detracting from it.
+
+<!--endintro-->
+
+When there are multiple items listed, it can be challenging to distinguish between them quickly, leading to confusion and miscommunication. If the same emoji is repeated multiple times within a list, it can create visual clutter and make the list more difficult to read. By using the DRY principle for emojis, you can create lists that are not only more visually appealing, but also more accessible and understandable for a wider audience. This can be especially helpful in professional or technical contexts, where clear communication is essential for success.
+
+::: greybox
+### ✅ Pros
+
+- ✅ Increases productivity
+- ✅ Saves time
+- ✅ Reduces stress
+
+### ❌ Cons
+
+- ❌ May be challenging to implement
+- ❌ May take time to adjust
+- ❌ Can be challenging to maintain
+
+:::
+::: bad
+Bad example – Using an excessive amount of emojis. Not following DRY :(
+:::
+
+
+::: greybox
+### ✅ Pros
+
+- Increases productivity
+- Saves time
+- Reduces stress
+
+### ❌ Cons
+
+- Requires effort to implement
+- May take time to adjust
+- Can be challenging to maintain
+:::
+::: good
+Good example – using the DRY principle 
+:::
+
+
+By avoiding repeated emojis within each category, the list remains clear and concise, which can improve communication and understanding. Thus making it easy to scan the list and understand the benefits and drawbacks of a particular situation. 
+
+Following the DRY principle by avoiding excessive repetition of emojis helps to create lists that are visually interesting and easy to read, while also promoting efficient and maintainable content creation.
+
+
+::: tip
```

Comment:
Hey @JackDevAU 

There is not box type "tip"
See https://ssw.com.au/rules/rule/#5-boxes 

1. Please fix
I would add just the prefix "Tip:" in bold in front of the sentence